### PR TITLE
SEO fixes for docs

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**

### CODE REVIEW

File: docs/api-reference/commit.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Commit
 
 ```eval_rst
+.. meta::
+   :description: Use Jovian commit to upload jupyter notebooks online
```
Comment (sidujjain) [inline docs/api-reference/commit.md:5]:
```suggestion
   :description: Use Jovian commit to upload and share Jupyter notebooks on Jovian platform.
```

File: docs/api-reference/oembed.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ Parameters:
 _Example URLs:_
 
 - Jovian Notebook URLs
-  - [https://jovian.ml/aakashns/01-pytorch-basics](https://jovian.ml/aakashns/01-pytorch-basics)
-  - [https://jovian.ml/aakashns/movielens-fastai/v/14](https://jovian.ml/aakashns/movielens-fastai/v/14)
+  - [Jovian Notebook](https://jovian.ml/aakashns/01-pytorch-basics)
+  - [Jovian Notebook (Version specified)](https://jovian.ml/aakashns/movielens-fastai/v/14)
 - Jovian Viewer URLs
-  - [http://jovian.ml/viewer?url={notebook-url}](http://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
+  - [https://jovian.ml/viewer?url={notebook-url}](https://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
```
Comment (sidujjain) [inline docs/api-reference/oembed.md:23]: add text in []

File: docs/api-reference/oembed.md [hunk 3]
```diff
@@ -39,3 +39,8 @@ Response:
   "html": "<iframe src='https://jovian.ml/embed?url=https%3A//jvn.storage.googleapis.com/gists/aakashns/5bc23520933b4cc187cfe18e5dd7e2ed/raw/901a9d2508bd441dbf06954c5f46bf58/movielens-fastai.ipynb' title='Jovian Viewer' height=800 width=800 frameborder=0 allowfullscreen></iframe>"
 }
 ```
+
+```eval_rst
+.. meta::
+   :description: Embed online jupyter notebooks to blogs and websites
```
Comment (sidujjain) [inline docs/api-reference/oembed.md:45]:
```suggestion
   :description: Embed Jupyter notebooks on blogs and websites.
```

File: docs/callbacks/fastai.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ## Fastai Callback
 
 ```eval_rst
+.. meta::
+   :description: Track hyperparameters, metrics with fastai callback by Jovian
```
Comment (sidujjain) [inline docs/callbacks/fastai.md:5]:
```suggestion
   :description: Track hyper-parameters and metrics with Fastai callback by Jovian.
```

File: docs/callbacks/keras.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 ## Keras Callback
 
 ```eval_rst
+.. meta::
+   :description: Track hyperparameters, metrics with keras callback by Jovian
```
Comment (sidujjain) [inline docs/callbacks/keras.md:5]:
```suggestion
   :description: Track hyper-parameters and metrics with Keras callback by Jovian.
```

File: docs/integrations/conda.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
 # Anaconda Integration
 
 Page under Construction
+
+```eval_rst
+.. meta::
+   :description: Jovian.ml integration with anaconda.org
```
Comment (sidujjain) [inline docs/integrations/conda.md:7]: Jovian integration with Anaconda. 

Let's use "Jovian" (instead of Jovian.ml) and punctuation at the end for consistency.

File: docs/user-guide/attach.md [hunk 7]
```diff
@@ -1,5 +1,10 @@
 ## Attaching files and model outputs
 
+```eval_rst
+.. meta::
+   :description: Attach files, images, utils, python to online Jovian jupyter notebooks
```
Comment (sidujjain) [inline docs/user-guide/attach.md:5]:
```suggestion
   :description: Attach files, images and model outputs to Jovian projects.
```

File: docs/api-reference/logger.md [hunk 8]
```diff
@@ -1,6 +1,8 @@
 ## Log Dataset, Hyperparams & Metrics
 
 ```eval_rst
+.. meta::
+   :description: Use Jovian to track hyperparameters, metrics and dataset.
```
Comment (sidujjain) [inline docs/api-reference/logger.md:5]: hyper-parameters
(change in other places too)

Comment (PrajwalPrashanth) [reply docs/api-reference/logger.md:5]: I think the general usage is `hyperparameters` including wikipedia. Would it be good for SEO, if we change it ?

Comment (sidujjain) [reply docs/api-reference/logger.md:5]: right. let's use hyperparameters then.

File: docs/api-reference/oembed.md (hunk 2 above)
Comment (PrajwalPrashanth) [reply docs/api-reference/oembed.md:23]: That label for the link is a example of the URL format to be used, think it can be retained as above. Let me know if it needs to be changed.

**Comment by PrajwalPrashanth** [comment]
**Update**
- Added missing period punctuation
- Changed instances of `online jupyter notebooks` --> `Jupyter notebooks` as per the latest data it has more traffic and low competition too.
- Changed instances of `Jovian.ml` --> `Jovian`

File: docs/user-guide/track.md [hunk 9]
```diff
@@ -1,5 +1,10 @@
 ## Tracking Datasets, Hyperparameters and Metrics
 
+```eval_rst
+.. meta::
+   :description: Track Datasets, Hyperparameters and Metrics for Jupyter notebooks.
```
Comment (sidujjain) [inline docs/user-guide/track.md:5]:
```suggestion
   :description: Track datasets, hyperparameters and metrics for ML experiments.
```

File: docs/user-guide/compare.md [hunk 10]
```diff
@@ -1,5 +1,10 @@
 ## Comparing and Analyzing experiments
 
+```eval_rst
+.. meta::
+   :description: Compare and analyze different versions of Jupyter notebooks with Jovian.
```
Comment (sidujjain) [inline docs/user-guide/compare.md:5]:
```suggestion
   :description: Compare and analyze different versions of ML experiments with Jovian.
```

File: docs/user-guide/collaborate.md [hunk 11]
```diff
@@ -1,5 +1,10 @@
 ## Collaborating on Jovian projects
 
+```eval_rst
+.. meta::
+   :description: Collaborate on Jupyter notebooks with Jovian platform.
```
Comment (sidujjain) [inline docs/user-guide/collaborate.md:5]:
```suggestion
   :description: Collaborate on Jupyter notebooks using Jovian platform.
```
